app/client/src/features/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { graphql, PreloadedQuery, usePreloadedQuery } from 'react-relay';
+import { useRouter } from 'next/router';
+import { isBefore, isAfter } from 'date-fns';
+
+import { Card, CardContent, Grid, List, Typography, IconButton } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { Add } from '@mui/icons-material';
```

Comment:
Generally, we only try to split up imports if one is local, e.g. `@local/...` and something from a package. Basically, you can remove the newline on line 5.